tools/run-tests-within-container

```diff
@@ -56,7 +58,5 @@ else
     run_test -r $list
 fi
 
-[ -r tests_failed ] && exit 1
-if test -n "$TRAVIS"; then
-    cp -a assets/cache/* /opt/openqa/assets/cache
-fi
+[ "$TESTS_FAILED_FLAG" != 0 ] && exit 1
```

Comment:
```suggestion
exit [ "$TESTS_FAILED_FLAG" != 0 ]
```

Reply:
I don't think is correct, maybe you want something like this

```suggestion
exit $(( TESTS_FAILED_FLAG != 0 ))
```

Reply:
why not `exit "$TESTS_FAILED_FLAG"`?

Reply:
Yes much better and less overengineered :100: